common/build.gradle

```diff
@@ -12,6 +12,7 @@ repositories {
 
 dependencies {
     api project(':api')
+    implementation 'org.projectlombok:lombok:1.18.26'
```

Comment:
Why add lombok?

Reply:
For some reason it just stopped wanting to compile until I added it, I can try removing it again and see if it works.

Reply:
Removed lombok and it still works, odd it broke in the first place.